src/goto-symex/shadow_memory_util.cpp

```diff
@@ -22,7 +22,6 @@ Author: Peter Schrammel
 #include <util/ssa_expr.h>
 #include <util/std_expr.h>
 
-
```

Comment:
This commit should be squashed into the respective previous commits.

Reply:
Agreed.

Changed